client/routing.js

```diff
@@ -1,49 +1,66 @@
-import Thimbleful from 'thimbleful';
-const router = new Thimbleful.Router();
+import Thimbleful from 'thimbleful'
 
-import patientOverview    from './components/patient-overview';
-import inbox              from './components/inbox';
-import transactions       from './components/transactions';
-import patient            from './components/patient/patient';
-import irmaLogin          from './components/irma-login';
-import remoteOrganisation from './components/patient/remote/organisation';
+import patientOverview from './components/patient-overview'
+import inbox from './components/inbox'
+import transactions from './components/transactions'
+import patient from './components/patient/patient'
+import irmaLogin from './components/irma-login'
+import logout from './components/logout'
+import remoteOrganisation from './components/patient/remote/organisation'
+import header from './components/header'
+
+const router = new Thimbleful.Router()
 
 export default {
   load: () => {
     // Root redirects to patient overview
-    if ( !window.location.hash ) window.location.hash = 'dashboard';
+    if (!window.location.hash) window.location.hash = 'dashboard'
 
     router.addRoute('dashboard', async link => {
-      await patientOverview.render();
-      openPage(link);
+      openPage('private', link)
+
+      // Render organisation name, colour and user
+      header.render()
```

Comment:
Ik weet niet of ik zo'n fan ben van overal `header.render()`.
Maar ik weet ook nog niet zo goed hoe dit soort dingen wel in mijn "niet-framework" passen. Ga ik eens over nadenken, tot die tijd is dit prima.

Reply:
Ik heb een idee hoe we dit kunnen tackelen. Ga ik mee spelen nadat deze branch gemerged is 👌🏻